UI/window-basic-settings.cpp

```diff
@@ -448,6 +512,7 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 	HookWidget(ui->advOutNoSpace,        CHECK_CHANGED,  OUTPUTS_CHANGED);
 	HookWidget(ui->advOutRecFormat,      COMBO_CHANGED,  OUTPUTS_CHANGED);
 	HookWidget(ui->advOutRecEncoder,     COMBO_CHANGED,  OUTPUTS_CHANGED);
+	HookWidget(ui->advOutRecAudioEncoder,     COMBO_CHANGED,  OUTPUTS_CHANGED);
```

Comment:
Again here:
`advOutRecAudioEncoder -> advOutRecAEncoder`
then make these parameters align with the others to cause us less physical pain.